CHANGELOG.md

```diff
@@ -1,161 +1,162 @@
+-	TableList.selectionEnabled deprecated since 4.29. Use selectionMode and dragEnabled instead.
+-	The addBookmark property within Bookmarks.visibleElements is deprecated at 4.29. Use visibleElements.addBookmarkButton instead.
+-	The basemap IDs referencing the basemap layer service v1 (i.e. arcgis-topographic) were deprecated at version 4.28. Support for these basemaps will be removed at version 4.31. Use basemaps from the basemap style service (v2) instead (i.e. arcgis/topographic).
+-	The "connectivity" possible value for QueryAssociationsParameters.types is deprecated at 4.29. Please use "junction-junction-connectivity" instead.
+-	TimeExtent deprecated since version 4.31. Use TimeExtent instead.
+-	TimeInterval deprecated since version 4.31. Use TimeInterval instead.
+-	TimeSlider.getPropertiesFromWebMap(webMap, signal).getPropertiesFromWebMap deprecated since 4.29. Use getTimeSliderSettingsFromWebDocument instead.
+-	TimeSlider.own deprecated since 4.28 Use addHandles() instead.
+-	TimeSliderViewModel.getPropertiesFromWebMap(webMap, signal).getPropertiesFromWebMap deprecated since 4.29. Use getTimeSliderSettingsFromWebDocument instead.
+-	timeUtils.getTimeSliderSettingsFromWebMap deprecated since 4.30. Use getTimeSliderSettingsFromWebDocument instead.
+-	TimeZoneLabel.own deprecated since 4.28 Use addHandles() instead.
+-	Track.own deprecated since 4.28 Use addHandles() instead.
+-	UtilityNetworkAssociations.own deprecated since 4.28 Use addHandles() instead.
+-	UtilityNetworkTrace.gdbVersion deprecated since 4.31, gdbVersion will be removed and the gdbVersion of the UtilityNetwork will be consumed directly.
+-	UtilityNetworkTrace.own deprecated since 4.28 Use addHandles() instead.
+-	UtilityNetworkTraceViewModel.gdbVersion deprecated since 4.31, gdbVersion will be removed and the gdbVersion of the UtilityNetwork will be consumed directly.
+-	UtilityNetworkValidateTopology.own deprecated since 4.28 Use addHandles() instead.
+-	ValidateNetworkTopologyResult.dirtyAreaCount deprecated since version 4.28. Dirty area count was implemented in the original version of utility network, but as of schema version 4 of the utility network, this is no longer supported.
+-	ValuePicker.own deprecated since 4.28 Use addHandles() instead.
+-	VersionManagementViewModel.versionIdentifierLookup deprecated since version 4.30. Use VersioningState instead.
+-	VersionManagementViewModel.versionInfoLookup deprecated since version 4.30. Use VersioningState instead.
+-	VersionManagementViewModel.versionManagementServiceLookup deprecated since version 4.30. Use VersioningState instead.
+-	VideoPlayer.own deprecated since 4.28 Use addHandles() instead.
+-	WCSLayer.rasterInfo deprecated since 4.29. Use serviceRasterInfo instead.
+-	Weather.own deprecated since 4.28 Use addHandles() instead.
+-	Widget.own deprecated since 4.28 Use addHandles() instead.
+-	Zoom.own deprecated since 4.28 Use addHandles() instead.
```

Comment:
the formatting is a little funky on these. maybe remove the extra spaces?

Reply:
Oh that's odd. I'll see what I can do...I don't see that spacing in my IDE.

<img width="450" alt="image" src="https://github.com/user-attachments/assets/22e4654d-898a-4d0a-bfd0-e13308cf6233">

Reply:
Maybe it's a github thing. In the page preview it looks okay?

<img width="422" alt="image" src="https://github.com/user-attachments/assets/38e0521c-e2a4-4df7-ad27-9d5a34f55006">

Reply:
nm, I figured it out. I'm fixing it now.

Reply:
Fixed!

Reply:
nice, looks good!